agent/config/config.go

```diff
@@ -355,6 +380,16 @@ const (
 	// defaultPolicyWorkerAckTimeout is the default time limit that a policy
 	// eval must be ACK'd.
 	defaultPolicyEvalAckTimeout = 5 * time.Minute
+
+	// defaultLockPath is the default path used for the lock that syncs the leader
+	// election.
+	defaultLockPath = "nomad-autoscaler/lock"
+	// defaultLockTTL is the default ttl used for the lock that syncs the leader
+	// election.
+	defaultLockTTL = 5 * time.Minute
+	// defaultLockDelay is the default lockDelay used for the lock that syncs the leader
+	// election.
+	defaultLockDelay = 5 * time.Minute
```

Comment:
This means an autoscaler "cluster" could lose leadership for 10mins before another takes over, which seems a little long. What about something like 60s and 30s?